src/utils/backend/WampClient.js

```diff
@@ -136,6 +135,7 @@ export default class WampClient {
   }
 
   onDisconnected = () => {
+    this.backoff.reset()
```

Comment:
Why did you move the backoff reset? How is it related to the channel change?

Reply:
When I try to reopen websocket connection, setTimeout don't worked properly, because on every reopen we have `backoff.reset()`. Here we call `this.open()` - https://github.com/ubergrape/grape-web-client/blob/master/src/utils/backend/WampClient.js#L73 and then it's trigger - https://github.com/ubergrape/grape-web-client/blob/master/src/utils/backend/WampClient.js#L121

I'm deleted that, in order to on each reopen setTimeout work properly (like try to reconnect each 100, 200, 400, 800, 1600ms). Previously it's tried to reconnect each 100ms  because on each `reopen` backoff reset to initial value.

Reply:
It's related to this issue, because if something went wrong on initial data load, user will see reconnectiong each 100ms (disco-mode enabled:D)

Reply:
Thanks for the detailed explanation!